test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -620,6 +620,11 @@ TEST(LiteInterpreterTest, GetRuntimeByteCodeVersion) {
       runtime_bytecode_version == caffe2::serialize::kProducedBytecodeVersion);
 }
 
+TEST(LiteInterpreterTest, GetRuntimeOpsAndInfo) {
+  auto runtime_ops = _get_runtime_ops_and_info();
+  AT_ASSERT(runtime_ops.size() > 2900);
```

Comment:
where does 2900 come from?

Reply:
I was getting the total ops in the runtime (without custom ops) to be around 3k so I just picked a number that felt unlikely for the total ops to ever go beneath. I could iterate over the ops in the map and make sure they have schemas as well, but I'm hesitant to depend on any specific op existing or its schema being a certain size.